source/include/marlin/XMLParser.h

```diff
@@ -63,7 +63,7 @@ namespace marlin{
    *       &lt;group name="TwoJetAnalysis"/&gt;
    *     &lt;/if&gt;
    *   &lt;/if&gt;
-   *   &lt;processor name="MyLCIOOutputProcessor"/&gt;  
+   *   &lt;processor name="MyLCIOOutputProcessor"/&gt; 
```

Comment:
You missed this one

Reply:
Just as a guideline if you go to the page where the diff is displayed 
https://github.com/iLCSoft/Marlin/pull/19/files
and you add to the URL `?w=1` this will display the PR without whitespace changes https://github.com/iLCSoft/Marlin/pull/19/files?w=1
The display on both URLs has to be the same.

Reply:
Done !

Reply:
Can you squash again please